Formula/qt5.rb

```diff
@@ -90,12 +100,32 @@ def install
       -qt-pcre
       -nomake tests
       -no-rpath
+      -pch
+      -system-proxies
+      -optimized-tools
     ]
 
     args << "-nomake" << "examples" if build.without? "examples"
 
-    args << "-plugin-sql-mysql" if build.with? "mysql"
-    args << "-plugin-sql-psql" if build.with? "postgresql"
+    if build.without? "plugins"
+      use_plugin = "qt"
+      args << "-qt-sql-sqlite"
+      inreplace "qtbase/configure" do |s|
+        s.gsub! "CFG_JPEG=plugin", "CFG_JPEG=yes"
+        s.gsub! "CFG_GIF=plugin", "CFG_GIF=yes"
```

Comment:
@steve3d no, I've never passed any configure options to Qt5. However, Mike's question was whether what you're trying to change with string replacement can instead be handled by passing options. From reading the configure script, it appears that upstream at least intends for the jpeg and gif plugin settings to be configurable with configure options not direct modification of the file. If that sometimes, or always, fails under certain conditions, then it would be a bug, which would need to be reported with an upstream issue link or upstream pull request link. Since you're proposing this as a non-default option, the stakes are rather low here, but nonetheless if we follow our own "rules," this string replacement is not acceptable as-is since it's patching a bug you say you've identifiied in the configure script, and yet that bug has not been reported as far as I am aware.